Copyboard/Views/Common/CBCollectionView.swift

```diff
@@ -19,7 +19,7 @@ class CBCollectionView: NSCollectionView {
 		super.init(frame: .zero)
 		backgroundColors = [.clear]
 		isSelectable = true
-		allowsMultipleSelection = false
+		allowsMultipleSelection = true // set to "true" to allow mass deletion
```

Comment:
This breaks functionality, copyboard only handles one thing at a time, for example, copying multiple items at once is not supported, so this will be confusing to the user, instead have "bulk deletion" be in the contextmenu